omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -72,12 +72,15 @@ public void initBinder(WebDataBinder binder) {
 	public User formBackingObject(WebRequest request, @RequestParam(required = false, value = "person_id") Integer personId) {
 		String userId = request.getParameter("userId");
 		User u = null;
-		try {
-			u = Context.getUserService().getUser(Integer.valueOf(userId));
-		}
-		catch (Exception ex) {
-			log.error("Error while getting user", ex);
+		if (!StringUtils.isEmpty(userId)) {
+			try {
+				u = Context.getUserService().getUser(Integer.valueOf(userId));
+			}
+			catch (Exception ex) {
+				log.error("Error while getting user", ex);
+			}
```

Comment:
This has nothing to do with this feature, I just noticed a stack trace in the logs when I clicked on the link to create a new user (opens this form with userId = null), so put a null check around it.